src/frontend/src/lib/components/send/SendReview.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { createEventDispatcher } from 'svelte';
+	import SendData from '$lib/components/send/SendData.svelte';
+	import Button from '$lib/components/ui/Button.svelte';
+	import ButtonBack from '$lib/components/ui/ButtonBack.svelte';
+	import ButtonGroup from '$lib/components/ui/ButtonGroup.svelte';
+	import ContentWithToolbar from '$lib/components/ui/ContentWithToolbar.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { OptionBalance } from '$lib/types/balance';
+	import type { OptionToken } from '$lib/types/token';
+
+	export let destination = '';
```

Comment:
Why do you need all those parameters? We use a `SendContext` in modals, that pattern won't work here?

Reply:
Only one of these props is coming from the context (token). This is how they look in `IcSendReview` in `main`:

<img width="676" alt="Screenshot 2024-10-07 at 16 44 53" src="https://github.com/user-attachments/assets/a594af23-2474-4047-952f-025fec39434b">

Also, since the component I implemented in this PR is placed in `$lib`, I assume we don't want to use a context from `$icp-eth` or other network folder and rather make it generic.

Reply:
@peterpeterparker @AntonioVentilii-DFINITY would it be fine with you if we introduce the send store for BTC after we have all related components in place? I am implementing the flow from bottom to top (first child components, then parent ones), so without having the whole components tree in place, I find it a bit problematic to introduce and use a new store. Plus, I want to unblock @lmuntaner as he needs btc send modals to update them with pending transactions stuff.

Reply:
When it comes to me, I would say: no. I think it's best if we get it right from the start because we already suffered having to rework the modals multiple times and I know it's quite an effort which can also lead to issues.

Reply:
I can give a hand later today if you wish btw.

Reply:
@peterpeterparker that would be great! My main questions are whether it's possible to setup such a store at this stage (where BtcSendTokenWizard doesn't exist yet) and do we want to move the related store files from `icp-eth` to `lib`?

Reply:
I guess I can start using context, even if doesn't exist yet (it will be added later in `BtcSendModal`) since the components I'm adding are not in-use yet. 

And for now, I can re-use store from `icp-eth` and we can decide later where do we want to move.

Reply:
I updated the component, please take a look.